npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater_spec.rb

```diff
@@ -469,6 +469,35 @@
             to raise_error(Dependabot::DependencyFileNotResolvable)
         end
       end
+
+      context "with a dependency version that can't be found" do
+        let(:files) { project_dependency_files("#{npm_version}/yanked_version") }
```

Comment:
Test case is from the yarn lockfile updater: https://github.com/dependabot/dependabot-core/blob/main/npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater_spec.rb#L62-L69